src/main/java/net/moonbyte/AnarchyUtils/AnarchyUtils.java

```diff
@@ -34,7 +36,7 @@ public void onEnable() {
         // This was used during the 1.16 update to allow access to the patched paper dupe.
         // This is now disabled on moonbyte and will remain disabled. Any future dupes
         // Should be added here.
-        if (ConfigurationUtils.getConfig().getBoolean("dupes.donkey-dupe-enabled")) {
+        if (ConfigUtil.Dupes_DonkeyDupeEnabled) {
```

Comment:
We should probably rename this to 'PiglinDupe' or remove PiglinDupe entirely since this is not in use anymore.

I'd suggest just renaming the setting, since other server owners may want these dupes.